### ISSUE TITLE
x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by m4t7** [talltechdude/hassio-addons#1]
The Addon Debug output shows the following issue.

`{"level":"error","ts":1601409498.0998852,"msg":"/auth Error while retreiving user info after successful login at the OAuth provider: Post \"https://home.private.local/auth/token\": x509: certificate signed by unknown authority"}`

I suggest that there is a missing Root CA in the Alpine Linux Image.

Currently there is no option to update the Certifcate via the folder /ssl or /share
The config file is missing the map section.

  
```
"map": [
    "ssl:ro",
    "share:rw"
  ],
```
Could you add this to your repo or do I need to fork?

Then I could move my ca cert from `/share` to -> `/usr/local/share/ca-certificates` and perform an certificate store reload via
`update-ca-certificates`

**Comment by talltechdude** [comment]
Hi @m4t7, glad to see someone using these addons!

Which addon are you having this problem with? Are you referring to the Vouch Addon? Happy to help you troubleshoot and add the extra config if you need. Are you using your own CA, or just need the `update-ca-certificates` run in the build for public or ACME certs?

**Comment by m4t7** [comment]
Hello @talltechdude,
I came across your repo while searching for custom Hassio addons, I didn't know Vouch before. 😄 
Since I use multiple applications I thought a unified SSO endpoint would be very handy, I could combine this with other projects.

I have my own CA, and I use a domain that is not on the public internet.

The log output above is from the Vouch Addon. 

I switched to the Vouch Addon Container via Portainer and found that I could not import my CA cert because there is no mount from the host folder /share.

**Comment by m4t7** [comment]
Hello again,
I have solved the CA problem with a trick.
Therefore I have temporarily installed **telegraf:latest** from docker hub and set the following mount points:

`/mnt/data/supervisor/addons/data/1fdd3664_vouch` 	`/vouch`
`/mnt/data/supervisor/share` 	`/share`

Then I moved my CA cert within telegraf container from `/share` to `/vouch`.
Then I changed to the vouch container and moved the certificate from `/data` to `/usr/local/share/ca-certificates`.

Afterwards I performed a certificate store reload with `update-ca-certificates`. Now its working great!

So I have no problem anymore, maybe you add the map section for others or they follow my workaround.
